tests/stubdata/output/jats_aj_158_4_139_fulltext_tags.json

```diff
@@ -0,0 +1,2 @@
+{"ajab3643bib21": "[<p>Fortuitously, the timing of the ALMA observations [...] The last full fade and revival cycle of the SEB took place in 20092011 (Fletcher et al. <xref ref-type='bibr' rid='ajab3643bib21'>2011</xref>, <xref ref-type='bibr' rid='ajab3643bib22'>2017a</xref>), where the word fading is used when the SEB loses its brown color and turns white [...]</p>]",
```

Comment:
To support the output format change (see my comments on jats.py and test_jats.py before you come here), you'll need to do some small edits here too:
1. The value should be an array of strings instead of a string, e.g. instead of 
`"[<p>Fortuitously, the...` 
you'll want to do: 
`["<p>Fortuitously, the...`. 
Make sure that the value for the second key here (line 2 here) is an array of 4 object (i.e. you fix the double quotes for all 4 items in the array)

2. All of the single quotes (`ref-type='bibr'`) need to be converted to escaped double quotes (`ref-type=\"bibr\"`) - this will make sure the assertEqual line in the unittest passes and also keeps the json interpreter happy.